styles.css

```diff
@@ -0,0 +1,140 @@
+:root {
+    --background-color:white;
+    --grid-border-color:rgb(143, 136, 136);
+    --grid-font-color:rgb(134, 134, 142);
+    --nav-font-size:30px;
+}
+*{
+    box-sizing: border-box;
+}
+body {
+    display: grid;
+    grid-template-rows: auto auto 1fr;
+    grid-template-columns: 200px 1fr;
+    height: 100vh;
+    margin: 0;
+    font-family: Arial, sans-serif;
+    
+}
+.calendar-header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    background-color: var(--background-color);
+    grid-row: 1;
+    grid-column: 1 / 3;
+    padding: 8px 16px;
+    border-bottom: var(--grid-border-color);
+    margin-bottom: 30px;
+    padding-top: 15px;
+    padding-bottom: 15px;
+    border-bottom:  1px solid var(--grid-border-color);
+}
```

Comment:
not sure if you are planning to come back to styles, but it would be nice that header would not hide when scrolling, and scroll would happen only in the calendar cells. : )